10x_Experimental_Scenarios.md

```diff
@@ -192,134 +175,432 @@ This CSV will be used to construct both [the sample sheet CSV input for `mkfastq
+S2 -|->  H1   -|->   L1-ADT   -|->   PL1  ->  BCL1  -|->  FQ1-ADT  -|->  FBM1  -|->  FBM1-S2 
+S3 -|          |->   L1-HTO   -|                     |->  FQ1-HTO  -|           |->  FBM1-S3 
+S4 -|                                                                           |->  FBM1-S4 
+```
+| FASTQs | From BCL | To Output |
+|---|---|---|
+| FQ1-GEX | BCL1 | FBM1 |
+| FQ1-ADT | BCL1 | FBM1 |
+| FQ1-HTO | BCL1 | FBM1 |
+
+This is the same as the default example used earlier in the documentation. Four ADT samples are hashed into a single Loading Sample (`H1`) and run in a single 10x lane, which generates three libraries. The libraries are pooled and sequenced once. The BCL files are de-multiplexed into three Seq-Run FASTQ Sets and one FBM FASTQ Set. Cell Ranger `count` (using feature barcoding) is run once to produce one FBM containing GEX, ADT, and HTO data. De-hashing is manually run after the Processing Run to obtain sample level FBMs. 
+
+## 10. Hashed-ADT, Multiple Lanes per Sample, ADT, Single Seq-Run
+```
+ Hash      Make             Pool         Seq Pooled     Demulti           Calc       De-hash 
+ Samples   Libraries        Libraries    Library        BCL               FBM        Samples
+ -------   ---------        ---------    ---------      ====              ====       --------
+              |->   L1-XL1-GEX   -|                     |->  FQ1-XL1-GEX  -|           |->  FBM1-S1 
+              |->   L1-XL1-ADT   -|                     |->  FQ1-XL1-ADT  -|->  FBM1  -|->  FBM1-S2 
+              |->   L1-XL1-HTO   -|                     |->  FQ1-XL1-HTO  -|           |->  FBM1-S3 
+S1 -|         |                   |                     |                              |->  FBM1-S4 
+S2 -|->  H1  -|                   |->   PL1  ->  BCL1  -|
+S3 -|         |                   |                     |
+S4 -|         |->   L1-XL2-GEX   -|                     |->  FQ1-XL2-GEX  -|           |->  FBM2-S1 
+              |->   L1-XL2-ADT   -|                     |->  FQ1-XL2-ADT  -|->  FBM2  -|->  FBM2-S2 
+              |->   L1-XL2-HTO   -|                     |->  FQ1-XL2-HTO  -|           |->  FBM2-S3 
+                                                                                       |->  FBM2-S4 
+```
+| FASTQs | From BCL | To Output |
+|---|---|---|
+| FQ1-XL1-GEX | BCL1 | FBM1 |
+| FQ1-XL1-ADT | BCL1 | FBM1 |
+| FQ1-XL1-HTO | BCL1 | FBM1 |
+| FQ1-XL2-GEX | BCL1 | FBM2 |
+| FQ1-XL2-ADT | BCL1 | FBM2 |
+| FQ1-XL2-HTO | BCL1 | FBM2 |
+
+Four ADT samples are hashed into a single Loading Sample (`H1`) and run in two 10x lanes (to get more cells), which generates six libraries. The libraries are pooled and sequenced once. The BCL files are de-multiplexed into six Seq-Run FASTQ Sets and two FBM FASTQ Set. Cell Ranger `count` (using feature barcoding) is run twice to produce two FBMs containing GEX, ADT, and HTO data. De-hashing is manually run twice after the Processing Run to obtain sample level FBMs. 
+
+# Glossary
```

Comment:
I'd like to make each term here a sub-heading so that we can link to them directly in the places in the doc where we say "see Glossary"; I'll send a PR for that separately though.

Reply:
I'll take care of it if you want

Reply:
ok, feel free to! or ping me when you want to start on it and i'll let you know if i already have